webui/react/src/hooks/usePermissions.ts

```diff
@@ -474,6 +476,14 @@ const canMoveProjectsTo = (
   );
 };
 
+const canMoveModel = (
+  { rbacAllPermission, rbacEnabled, userAssignments, userRoles }: RbacOptsProps,
+  destination?: PermissionWorkspace,
+): boolean => {
+  const destPermit = relevantPermissions(userAssignments, userRoles, destination?.id);
+  return rbacAllPermission || !rbacEnabled || destPermit.has(V1PermissionType.CREATEMODELREGISTRY);
+};
+
```

Comment:
@mapmeld is it sufficient?

Reply:
On other model permissions we are passing and checking `rbacModel` to check if the feature flag is on.

We might remove the feature flag in this release? I am checking on Slack

Reply:
Re: permissions checked in this function, I think this is good. Permission to move a model **from** a workspace can be checked in the UI there

Reply:
You should be able to move when rbac is off or if rbac is on and you have the correct permission.